### ISSUE TITLE
Cannot generate imports for p5.js - unsupported NamedDeclaration kind: 162

### ISSUE THREAD

**Issue by bartsokol** [fable-compiler/ts2fable#331]
I'm trying to generate fable imports for p5.js library and I'm constantly getting errors as below:

```ts2fable node_modules/@types/p5/index.d.ts src/Import.p5js.fs -e p5         ts2fable 0.8.0-build.494
export node_modules/@types/p5/constants.d.ts
export node_modules/@types/p5/global.d.ts
export node_modules/@types/p5/index.d.ts
export node_modules/@types/p5/literals.d.ts
export node_modules/@types/p5/lib/addons/p5.dom.d.ts
export node_modules/@types/p5/lib/addons/p5.sound.d.ts
export node_modules/@types/p5/src/color/creating_reading.d.ts
export node_modules/@types/p5/src/color/p5.Color.d.ts
export node_modules/@types/p5/src/color/setting.d.ts
export node_modules/@types/p5/src/core/constants.d.ts
export node_modules/@types/p5/src/core/environment.d.ts
export node_modules/@types/p5/src/core/p5.Element.d.ts
export node_modules/@types/p5/src/core/p5.Graphics.d.ts
export node_modules/@types/p5/src/core/p5.Renderer.d.ts
export node_modules/@types/p5/src/core/rendering.d.ts
export node_modules/@types/p5/src/core/structure.d.ts
export node_modules/@types/p5/src/core/transform.d.ts
export node_modules/@types/p5/src/data/local_storage.d.ts
export node_modules/@types/p5/src/data/p5.TypedDict.d.ts
export node_modules/@types/p5/src/events/acceleration.d.ts
export node_modules/@types/p5/src/events/keyboard.d.ts
export node_modules/@types/p5/src/events/mouse.d.ts
export node_modules/@types/p5/src/events/touch.d.ts
export node_modules/@types/p5/src/image/image.d.ts
export node_modules/@types/p5/src/image/loading_displaying.d.ts
export node_modules/@types/p5/src/image/p5.Image.d.ts
export node_modules/@types/p5/src/image/pixels.d.ts
export node_modules/@types/p5/src/io/files.d.ts
export node_modules/@types/p5/src/io/p5.Table.d.ts
export node_modules/@types/p5/src/io/p5.TableRow.d.ts
export node_modules/@types/p5/src/io/p5.XML.d.ts
export node_modules/@types/p5/src/math/calculation.d.ts
export node_modules/@types/p5/src/math/math.d.ts
export node_modules/@types/p5/src/math/noise.d.ts
export node_modules/@types/p5/src/math/p5.Vector.d.ts
export node_modules/@types/p5/src/math/random.d.ts
export node_modules/@types/p5/src/math/trigonometry.d.ts
export node_modules/@types/p5/src/typography/attributes.d.ts
export node_modules/@types/p5/src/typography/loading_displaying.d.ts
export node_modules/@types/p5/src/typography/p5.Font.d.ts
export node_modules/@types/p5/src/utilities/array_functions.d.ts
export node_modules/@types/p5/src/utilities/conversion.d.ts
export node_modules/@types/p5/src/utilities/string_functions.d.ts
export node_modules/@types/p5/src/utilities/time_date.d.ts
export node_modules/@types/p5/src/webgl/3d_primitives.d.ts
export node_modules/@types/p5/src/webgl/interaction.d.ts
export node_modules/@types/p5/src/webgl/light.d.ts
export node_modules/@types/p5/src/webgl/loading.d.ts
export node_modules/@types/p5/src/webgl/material.d.ts
export node_modules/@types/p5/src/webgl/p5.Camera.d.ts
export node_modules/@types/p5/src/webgl/p5.Geometry.d.ts
export node_modules/@types/p5/src/webgl/p5.RendererGL.d.ts
export node_modules/@types/p5/src/webgl/p5.Shader.d.ts
export node_modules/@types/p5/src/core/shape/2d_primitives.d.ts
export node_modules/@types/p5/src/core/shape/attributes.d.ts
export node_modules/@types/p5/src/core/shape/curves.d.ts
export node_modules/@types/p5/src/core/shape/vertex.d.ts
unsupported Statement kind: 254
unsupported Statement kind: 249
createIExportsModule [p5/constants], constants
unsupported Statement kind: 254
unsupported Statement kind: 249
createIExportsModule [p5/global], global
unsupported NamedDeclaration kind: 162
Error: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
```

I tried it with both current (0.7.0) and next versions and getting the same error.

**Comment by slang25** [comment]
I'm seeing the same thing, it was succeeding on 0.6.2.
For me I'm running it against https://github.com/microsoft/playwright
with the following command:
```
ts2fable index.d.ts index.fs -e PlaywrightWeb Page
```

Output:
```
ts2fable 0.7.0
export lib/chromium/crPage.d.ts
export lib/firefox/ffPage.d.ts
export lib/webkit/wkPage.d.ts
export lib/webkit/wkPageProxy.d.ts
export lib/webkit/wkProvisionalPage.d.ts
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported Statement kind: 254
unsupported TypeNode kind: 216
Error: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
```

**Comment by humhei** [comment]
Should be fixed in 0.7.1
